nixos/modules/programs/trippy.nix

```
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, ... }:
```

```suggestion
{ lib, config, pkgs, ... }:
```

https://nixos.org/manual/nixpkgs/unstable/#chap-conventions

> Arguments should be listed in the order they are used, with the exception of lib, which always goes first.